### PR TITLE
Config settings

### PR DESCRIPTION
**Pull request by MajinBui**
Added configuration settings to the main file and a set time function.  However, since the plugin doesn't 'activate', I couldn't get the settings to show up properly.

https://discuss.atom.io/t/configuration-vars-dont-show-up/14480

### CODE REVIEW

**Comment by ChinmayPatel** [comment]
Try pulling latest master. I have removed the activationCommands from package.json https://github.com/apigarage/atom-pomodoro-app/commit/7be5e3876c519dee57b2fed30d689de82d52b184#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L8. Now, the package should be loaded on start of Atom.

**Comment by MajinBui** [comment]
Hmm. Something strange is occurring. Upon further investigation, the settings view isn't working correctly with pomodoro. 
https://github.com/MajinBui/atom-pomodoro-app/blob/config-settings/lib/index.coffee#L5-L40
When these lines are simply copied into the ascii-art package for testing, they show up in the settings menu for it.
capture1

capture2

However, it does not appear anywhere in atom's section.  In fact, not even the ReadMe file is being read.
capture3

**Comment by ChinmayPatel** [comment]
Seems like, your solution is working. haven't tried it on my machine, yet. This is another crazy week for me, but will look into it again on next weekend. #crossingmyfingers.

**Comment by MajinBui** [comment]
I'll play around with the base code as well, but otherwise I wish you good luck.

**Comment by MajinBui** [comment]
I fixed it.  The issue was a on the package.json file where the "name" key had to be exactly the same as the project folder name.  Always the small typos that get you.

File: lib/index.coffee [hunk 1]
```diff
@@ -2,10 +2,51 @@ PomodoroAppView = require './pomodoro-app-view'
...
+    startTime:
+      title: 'Start Time'
+      type: 'object'
+      properties:
+        minute:
+          type: 'integer'
+          default: 25
+          minimum: 0
+          maximum: 60
+        second:
```
Comment (apigarage) [inline lib/index.coffee]: I was wondering, if we need seconds or not? We can default to 00 everytime. That will also simplify our user experience. What do you think?

**Comment by apigarage** [comment]
That's a very good catch. I was trying to follow a guideline where they ask us to not use atom in their plugin names. May be, we can rename things later on.

The code is perfect. Just one comment regarding: do we need seconds in the settings or not? IMO, we don't, but I am okey if you have strong preferences for seconds.

Happy to merge the code as soon as you get back. Well done :) :).

**Comment by MajinBui** [comment]
I'd prefer it without seconds as well haha. I'll change it up in a few.

**Comment by MajinBui** [comment]
Done.

**Pull request merged**